docs/projects/_configs.ts

```diff
@@ -0,0 +1,42 @@
+export interface Project {
+  id: string
+  goal_statement: string
+  emoji: string
+  project_tags: string[]
+  headline: string
+  description: string
+  level_of_difficulty: 'Beginner' | 'Intermediate' | 'Advanced'
+  roles: string[]
+}
+
+export interface ProjectsConfig {
+  [key: string]: Project
+}
```

Comment:
change types to below

```suggestion
type LevelOfDiffulty = 'Beginner' | 'Intermediate' | 'Advanced';

export type Project = {
  id: string;
  goal_statement: string;
  emoji: string;
  project_tags: string[];
  headline: string;
  description: string;
  level_of_difficulty: LevelOfDiffulty;
  roles: string[];
};
```